pkg/vm/stackitem/serialization.go

```diff
@@ -210,6 +210,10 @@ func decodeBinary(r *io.BinReader, allowInvalid bool) Item {
 		return NewBigInteger(num)
 	case ArrayT, StructT:
 		size := int(r.ReadVarUint())
+		if size > MaxArraySize {
```

Comment:
Are you sure that this constraint is still implemented in Neo? I was able to find only the `MaxStackSize=2*1024` constraint. Could you provide a link to the line in C# code, please?

Reply:
Looks like it's completely gone since preview2. Unbelievable.

Reply:
Fixed.